src/problems/richardsmitchellettesudoku.clj

```diff
@@ -0,0 +1,58 @@
+(ns problems.richardsmitchellettesudoku)
+
+(defrecord State [board])
+
+(defn getSection [board box] (nth board (int (/ box 3))))
+(defn getBox [board box] (nth (nth board (int (/ box 3))) (rem box 3)))
+
+(defn getBoxRow [board box boxRow] (nth
+                               (nth
+                                 (nth board (int (/ box 3)))
+                                    (rem box 3))
+                                      boxRow))
+
+(defn getByBox [board box boxRow boxCol] (nth
+                                           (nth
+                                             (nth
+                                               (nth board (int (/ box 3)))
+                                             (rem box 3))
+                                           boxRow)
+                                         boxCol))
+(defn getByRowCol [board row col] (getByBox board (+ (* 3 (int (/ row 3))) (int (/ col 3))) (rem row 3) (rem col 3)))
```

Comment:
You've got a lot of *really* long lines in this file. A few line breaks would definitely help make things more readable. Also, whenever you have long lines/functions, that's a good place to see if there might be a way you could break things up to improve the structure. (Although, to be fair, you created quite a few little helper methods.)